docs/sphinx/sections/api/apidocs/partitions.rst

```diff
@@ -46,6 +46,13 @@ Partition Mapping
 
 .. autoclass:: StaticPartitionMapping
 
+.. autoclass:: SpecificPartitionsPartitionMapping
```

Comment:
cc @clairelin135  is there an order of these partition mappings that would be helpful? like to any of them build on each other?

Reply:
The MultiPartitionMapping and MultiToSingleDimensionPartitionMapping do build on other ones, I think it's good that they're listed the last